benchmark/utils.go

```diff
@@ -20,16 +20,87 @@ import (
 	"encoding/csv"
 	"encoding/json"
 	"errors"
+	"fmt"
 	"io"
 	"os"
 	"os/exec"
+	"time"
+
+	"github.com/containerd/containerd"
+	"github.com/containerd/containerd/contrib/nvidia"
+	"github.com/containerd/containerd/oci"
+	runtimespec "github.com/opencontainers/runtime-spec/specs-go"
 )
 
+const testContainerID = "TEST_RUN_CONTAINER"
+
 type ImageDescriptor struct {
-	ShortName            string `json:"short_name"`
-	ImageRef             string `json:"image_ref"`
-	SociIndexManifestRef string `json:"soci_index_manifest_ref"`
-	ReadyLine            string `json:"ready_line"`
+	ShortName            string       `json:"short_name"`
+	ImageRef             string       `json:"image_ref"`
+	SociIndexManifestRef string       `json:"soci_index_manifest_ref"`
```

Comment:
unrelated nit: Can we rename this to `SociIndexManifestDigest` or `SociIndexDigest`?